cockatrice/src/client/ui/widgets/visual_deck_storage/deck_preview/deck_preview_widget.cpp

```diff
@@ -226,14 +225,23 @@ void DeckPreviewWidget::updateBannerCardComboBox()
         return a.first.toLower() < b.first.toLower();
     });
 
+    // This is *slightly* more performant than using addItem in a loop.
+
+    QStandardItemModel *model = new QStandardItemModel(pairList.size(), 1, bannerCardComboBox);
+
+    int row = 0;
     for (const auto &pair : pairList) {
         QVariantMap dataMap;
         dataMap["name"] = pair.first;
         dataMap["uuid"] = pair.second;
 
-        bannerCardComboBox->addItem(pair.first, dataMap);
+        QStandardItem *item = new QStandardItem(pair.first);
+        item->setData(dataMap, Qt::UserRole);
```

Comment:
Turns out you're allowed to directly wrap QPair in QVariant, which would save on creating a `QVariantMap`. Would doing that improve performance, or is it too minor to matter? 
```suggestion
        item->setData(QVariant::fromValue(pair), Qt::UserRole);
```

Reply:
![image](https://github.com/user-attachments/assets/ef3f46db-3c3c-4340-a5ed-b206f1166564)
![image](https://github.com/user-attachments/assets/f22f70f5-0ee4-4081-ba7d-57e221260b92)

We shave a little off. We may as well. Good find :)